sdks/python/apache_beam/io/gcp/bigquery_tools_test.py

```diff
@@ -224,9 +224,15 @@ def test_delete_dataset_retries_for_timeouts(self, patched_time_sleep):
     self.assertTrue(client.datasets.Delete.called)
 
   @mock.patch('time.sleep', return_value=None)
+  @mock.patch(
+      'apitools.base.py.base_api._SkipGetCredentials', return_value=True)
   @mock.patch('google.cloud._http.JSONConnection.http')
-  def test_user_agent_insert_all(self, http_mock, patched_sleep):
-    wrapper = beam.io.gcp.bigquery_tools.BigQueryWrapper()
+  def test_user_agent_insert_all(
+      self, http_mock, patched_skip_get_credentials, patched_sleep):
+    try:
```

Comment:
This does not sounds a valid workaround. It's like error happens in a line and then we skip the test. If it is "it needs network access to get credential.", then we should skip the test when there isn't network access in the decorator, and not inside the test

Reply:
Ok. Looks like that google.cloud.bigquery is not accessible in the internal test environment. I just made some change on the code. PTAL.